extensions/sql-migration/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "sql-migration",
   "displayName": "%displayName%",
   "description": "%description%",
-  "version": "1.2.3",
+  "version": "1.2.4",
```

Comment:
need to bump this up. i just updated it

Reply:
yeah, will follow up on this after your release is done